versioned_docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -157,6 +166,15 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```

Comment:
As there are 2 different chapters between Engine and Legacy, I will only mention the Legacy package here.